test/index.test.js

```diff
@@ -279,6 +280,24 @@ describe('Segment.io', function() {
         analytics.assert(object.context);
         analytics.assert(!object.context.amp);
       });
+
+      it('should add a list of enabled integrations when `addEnabledMetadata` is set', function() {
+        var opts = extend({ addEnabledMetadata: true }, options);
+        var Analytics = analytics.constructor;
+        var ajs = new Analytics();
+        ajs.use(Segment);
+        ajs.use(integration('other'));
+        ajs.initialize({ 'Segment.io': opts, other: {} });
```

Comment:
The tests set up everything using a standard set of options. We could hack on the extra option here (with `segment.options.addEnabledMetadata = true`), but this is what other cases did when they needed things initialized with a custom option.